lib/components/ms-stepper/MsWizardStepper.vue

```diff
@@ -12,7 +12,7 @@
       />
       <ion-text
         class="subtitles-sm step-title"
-        :class="index < currentIndex ? MsStepStatus.DONE : ''"
+        :class="index < currentIndex ? MsStepStatus.DONE : index === currentIndex ? MsStepStatus.ACTIVE : MsStepStatus.DEFAULT"
```

Comment:
Double ternaries are impossible to read, use the `{ 'class': condition }` syntax instead.